tests/wpunit/Tribe/Events/QueryTest.php

```diff
@@ -552,4 +563,63 @@ public function should_correctly_add_events_to_main_query_for_postst(): void {
 		$this->assertFalse( $wp_the_query->tribe_is_event_organizer );
 		$this->assertFalse( $wp_the_query->tribe_is_event_query );
 	}
+
+	/**
+	 * Validates that we inject our post type for tags query in the correct scenarios.
+	 *
+	 * @test
+	 */
+	public function should_filter_on_tags_archive() {
+		/**
+		 * @var WP_Query $wp_query
+		 */
+		global $wp_query;
+		// Store old wp query for reset later
+		$old_query = clone $wp_query;
+		// Given not on a particular page
+		$wp_query = new WP_Query( [] );
+		// Test terms
+		$show_slugs = [ 'transformers', 'gi-joe', 'he-man' ];
+		foreach ( $show_slugs as $show ) {
+			$term = get_term_by( 'slug', $show, 'tribe_events_cat' );
+			if ( ! $term ) {
+				wp_insert_term(
+					ucwords( str_replace( '-', ' ', $show ) ), // the term name
+					'tribe_events_cat', // the taxonomy
+					array(
+						'slug' => $show // the term slug
+					)
+				);
+			}
+		}
+
+		$tax_query = array('relation' => 'AND');
+		$tax_query[] = array(
+			'taxonomy' => 'post_tag',
+			'field' => 'slug',
+			'terms' => $show_slugs
+		);
+		$args = array(
```

Comment:
Please convert array() to [] and align the items correctly similar to [this](https://github.com/the-events-calendar/the-events-calendar/pull/4180/files#r1150873090).